ui/push-health/Health.jsx

```diff
@@ -229,17 +227,20 @@ export default class Health extends React.PureComponent {
           />
           {!!tests && !!currentRepo && (
             <React.Fragment>
-              <div>{percentComplete}% Complete</div>
-              <StatusProgress counts={status} />
+              <div className="d-flex mb-5">
+                <StatusProgress counts={status} />
+                <div>
```

Comment:
In the interest of centering the commit area around this a little better, let's add `className="mt-4 ml-2"` to this `<div>`